app/poros/author.rb

```diff
@@ -0,0 +1,3 @@
+class Author
```

Comment:
what were your thoughts for this file?

Reply:
If I made it to the extension I was going to create an author poro for author objects